index.html

```diff
@@ -0,0 +1,497 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Smashing magazine</title>
+  <link rel="stylesheet" href="assets/css/style.css">
+</head>
+
+<body class="total-width">
+  <header class="d-flex header">
+    <div class="logo-div bg-dark">
+
+    </div>
+    <div class="header-list d-flex">
+      <div class="list-item bg-medium"></div>
+      <div class="list-item bg-medium"></div>
+      <div class="list-item bg-medium"></div>
+      <div class="list-item bg-medium"></div>
+      <div class="list-item bg-medium"></div>
+      <div class="list-item bg-medium"></div>
+      <div class="list-item bg-medium"></div>
+      <div class="list-item bg-medium"></div>
+      <div class="list-item bg-dark"></div>
+    </div>
+    <div class="header-search bg-dark">
+
+    </div>
```

Comment:
- [ ] This should use ```<nav>``` semantic HTML. Create it using ```<ul>```, be aware of the elements that need to are links and use the link HTML tag for them.